pytest_django/plugin.py

```diff
@@ -44,6 +45,10 @@ def pytest_addoption(parser):
                      action='store_true', dest='create_db', default=False,
                      help='Re-create the database, even if it exists. This '
                           'option will be ignored if not --reuse-db is given.')
+    group._addoption('--xdist-one-db',
+                     dest='xdist_one_db', default=False, action='store_true',
+                     help="Use only one database with xdist plugin. "
+                          "Doesn't work with sqlite3 backend due to db lock")
```

Comment:
Oh, this implies it is true for sqlite in general - but should have a dot at the end.